### ISSUE TITLE
Debug kit not showing from version 3.10 (without error)

### ISSUE THREAD

**Issue by Bzounourz** [cakephp/debug_kit#538]
Hello,

When I update debug_kit to version 3.10.x, it no longer appears.

No error in php or cake logs.

No errors and no loading into the Google Chrome console.

Already tried:
-Deactivate and reactivate the sqlite and pdo_sqlite extensions (when disabled, I have an error in the logs, if activated, no error, so I guess it's good on that side)
-Delete the cache and empty the / tmp folder
-Execute bin / cake plugin assets symlink
-Create a "debug_kit" connection in mysql in the config / app file
-Delete debug_kit and reinstall it.

In short, I do not know what to do.

If I go back to version 3.9.5, it works again.

Ideas ?

Thank you in advance.

Environment:
CakePHP 3.4.9
PHP 7.0.10 or 5.6.25
WAMP on Windows 10

**Comment by ravage84** [comment]
Have you tried with a vanilla CakePHP  3.x app?

**Comment by Bzounourz** [comment]
I don't know what is a vanilla CakePHP app?
I found nothing on Google.

**Comment by cleptric** [comment]
@Bzounourz I guess @ravage84 meant to create a new cakephp app via `composer create-project cakephp/app` 🙂
You typically speaking of "vanilla" if you're using a program without any thrid party dependencies.
For example, if you play Minecraft without any mods, you would say Minecraft vanilla

**Comment by Bzounourz** [comment]
Thanks for the explanation!

I'll do that and come back to you.

**Comment by cesar128** [comment]
Ran into the same Problem... just installed a new version and "DebugKit is loaded." on homepage. still no debug bar showing up

**Comment by Tobse** [comment]
I am getting a "Missing Controller"-Exception in Cake\Routing\Exception\MissingControllerException.
"Error: Your routing resulted in Debug_kit as a controller name.
The controller name Debug_kit has not been properly inflected, and could not be resolved to a controller that exists in your application.
Ensure that your URL /debug_kit/toolbar/c024b3b6-ba4c-470d-b963-de86103aee80 is using the same inflection style as your routes do. By default applications use DashedRoute and URLs should use - to separate multi-word controller names."
CakePHP 3.5.8

Switching back to DebugKit 3.9 solves the problem.

**Comment by ADmad** [comment]
@Tobse your issue sounds similar to https://github.com/cakephp/debug_kit/issues/564#issuecomment-348625432

**Comment by arilia** [comment]
I have a similar problem. Started from a fresh installation v 3.6.0 I don't see the the debugKit toolbar. No php error, no messages in chrome console.

Switching back to debugKit 3.14 solves the problem

**Comment by arilia** [comment]
Seems that the problem arises in the isEnabled method. I have to understand why it return false.

**Comment by Teonnyn** [comment]
Running into the same thing as @arilia - Fresh 3.6 install, after running an upgrade and having DebugKit vanish, but no DebugKit (Loaded) and no error messages visible in any of the logs

**Comment by saeideng** [comment]
@Tobse  are you using on `localhost/yoursitename` domain ?

**Comment by andraskende** [comment]
@arilia @Teonnyn 
Same happened for me today on fresh 3.6 install..
It was the isSuspiciouslyProduction() so I added forceEnable to bootstrap.php

https://github.com/cakephp/debug_kit/blob/master/src/ToolbarService.php#L98

config/bootstrap.php

    if (Configure::read('debug')) {
        Configure::write('DebugKit', ['forceEnable' => true]);
        Plugin::load('DebugKit', ['bootstrap' => true]);
    }

**Comment by Teonnyn** [comment]
@andraskende That did the trick. Seems to be a bit of a weird function to have in Cake, though.

**Comment by saeideng** [comment]
@Teonnyn 
this behavior added in debug_kit 3.15 
see https://github.com/cakephp/debug_kit/pull/595

**Comment by davidyell** [comment]
Refs #611

**Comment by hsoberon** [comment]
@andraskende Thanks. This work for me. The forceenable did the trick. Thanks.
